pkg/kubectl/namespace_test.go

```diff
@@ -59,3 +59,28 @@ func TestNamespaceGenerate(t *testing.T) {
 		}
 	}
 }
+
+func TestNamespaceGenerateNoName(t *testing.T) {
+	tests := []struct {
+		params    map[string]interface{}
+		expected  *api.Namespace
+		expectErr bool
+	}{
+		{
+			params: map[string]interface{}{
+				"name": 1,
+			},
+
+			expectErr: true,
+		},
+	}
+	generator := NamespaceGeneratorV1{}
+	for _, test := range tests {
```

Comment:
This looks really similar to the existing test struct and usage.  Any chance you can refactor that to to handle this case?